### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by jellywiz**
# [Awesome books: with ES6](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            <button id="add-btn" type="button">Add</button>
+          </div>
+        </form>
+      </section>
+      <section class="contact-section page-sections">
+        <h2 class="header">Contact Information</h2>
+        <hr />
+        <div class="contact-section-wrapper">
+          <p>
+            Do you have any questions or whant to say "Hello"?
```
Comment (GiuseppeTG) [inline index.html]: - [OPTIONAL] Ups! Typo in this sentence, "whant" 😅

File: index.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import {
+  // eslint-disable-next-line max-len
+  inputTitle, inputAuthor, btn, listLink, addLink, contactLink, booksSection, addBookSection, contactSection,
```
Comment (GiuseppeTG) [inline index.js]: - [OPTIONAL] Please break this line of code into two lines of code. This will fix the problem without having to disable linters. 🤓

**Review by GiuseppeTG — APPROVED** [review]
Hi @jellywiz  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥇 : 
Congratulations! 🎉 

Alt Text

## Highlights
✅ I think your project is very well organized using modules. 
✅ App works fine.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (jellywiz) [reply index.html]: @GiuseppeTG Thanks for the insight, I will take care of it 
TyEeeGIF

File: index.js (hunk 2 above)
Comment (jellywiz) [reply index.js]: Thanks for the feedback @GiuseppeTG 😊

**Pull request merged**